cmd/snap/cmd_sign.go

```diff
@@ -47,7 +51,12 @@ func init() {
 	}, map[string]string{
 		// TRANSLATORS: This should not start with a lowercase letter.
 		"k": i18n.G("Name of the key to use, otherwise use the default key"),
-	}, nil)
+	}, []argDesc{{
+		// TRANSLATORS: This needs to begin with < and end with >
+		name: i18n.G("<filename>"),
```

Comment:
given you have this here, you don't need `positional-arg-name` above (and this is the right one to have, as `<filename>` should be translated IMO)

Reply:
> Thank you! A little nit, 

Hope i get more sleep the next days ;-)

> Please don't force-push from here on.

I know. 
Make it problem's to force in PRs on github ? I don't think so.

> We'll be adding tests to this; thank you for the change.
Have a lot to do. But can also write the test.